csharp/private/wrappers/dotnet.cc

```diff
@@ -0,0 +1,104 @@
+#include <iostream>
+#include <string>
+#include <sstream>
+
+#ifdef _WIN32
+#include <windows.h>
+#include <process.h>
+#include <errno.h>
+#else // not _WIN32
+#include <stdlib.h>
+#include <unistd.h>
+#endif // _WIN32
+
+#include "tools/cpp/runfiles/runfiles.h"
+
+using bazel::tools::cpp::runfiles::Runfiles;
+
+std::string evprintf(std::string name, std::string path)
```

Comment:
Can you run this through clang-format with the Google style rules?

Reply:
I have added `.clang-format` to the wrappers/ directory so that any new wrappers will follow the format guidelines.